darts/models/forecasting/regression_ensemble_model.py

```diff
@@ -219,6 +217,12 @@ def extreme_lags(
         extreme_lags_ = super().extreme_lags
         return (extreme_lags_[0] - self.train_n_points,) + extreme_lags_[1:]
 
+    @property
+    def supports_multivariate(self) -> bool:
+        return super().supports_multivariate and getattr(
+            self.regression_model, "supports_multivariate", True
+        )
```

Comment:
I'm not sure that they have that attribute? And I thought that `self.regression_model` could also be a sklearn model, but that is apparently not the case.

Reply:
`RegressionEnsembleModels` necessarily have this attribute, and it will be a darts `RegressionModel` wrapping around a sklearn model. Hence, the `support_multivariate()` will be available (and should be `True` thanks to the `MultiOutputModel` extending the univariate model to support multivariate).

Reply:
Okay, I implemented the change. It just makes sense to simplify it 👍